tests/unit/template_cassdc_test.go

```diff
@@ -302,9 +302,11 @@ var _ = Describe("Verify CassandraDatacenter template", func() {
 		It("enabling only medusa with local storage", func() {
 			options := &helm.Options{
 				SetValues: map[string]string{
-					"medusa.enabled": "true",
-					"medusa.storage": "local",
-					"reaper.enabled": "false",
+					"medusa.enabled":                 "true",
```

Comment:
Tests should be updated to verify that the volume mounts are added in the `medusa` and `medusa-restore` containers.